bip-update-process.md

```diff
@@ -0,0 +1,646 @@
+proposal is said to have rough consensus if it has been open to discussion on the development mailing list for at least
+one month, and no person maintains any unaddressed substantiated objections to it. Addressed or obstructive objections
+may be ignored/overruled by general agreement that they have been sufficiently addressed, but clear reasoning must be
+given in such circumstances. Active Process BIPs may be modified indefinitely as long as a proposed modification
+has rough consensus per the same criteria.[^living-documents]
+
+##### Revisions
+
+To help implementers understand updates to a BIP, any changes after it has reached Proposed or Active are
+tracked with version, date, and description in a Changelog section. The version number is inspired by semantic
+versioning (MAJOR.MINOR.PATCH). The MAJOR version is incremented if changes to the BIP’s Specification are introduced
+that are incompatible with prior versions (which should be an exception after a BIP is Proposed, and not happen to a
+BIP that is Active). The MINOR version is incremented whenever the specification of the BIP
+is changed or extended in a backward-compatible way. The PATCH version is incremented for other changes to the BIP that
+are noteworthy (bug fixes, test vectors, important clarifications, etc.). The version should start at 1.0.0 with the
+date the BIP was promoted to Proposed. An example of a Changelog section can be seen in
+[BIP 352](https://github.com/bitcoin/bips/blob/master/bip-0352.mediawiki). After a BIP receives a Changelog, the
+Preamble must indicate the latest version in the Revision header.
+
+#### Abandoned[^abandoned]
+
+A BIP that is _not in active use_, may be labeled Abandoned when its author has stopped working on it, no longer
+recommends the proposed approach, or no longer pursues adoption of their completed proposal. The reason for moving the
+proposal to Abandoned should be recorded in the Changelog section in the same commit that updates the status.
+
+##### Draft ↦ Abandoned
+
+BIP authors may decide on their own to change their BIP’s status from Draft to Abandoned. If a Draft BIP stops making
+progress, sees accumulated feedback unaddressed, or otherwise appears stalled for a year, the community may move the BIP
+to Abandoned unless the authors assert that they intend to continue work when contacted.
+
+##### Proposed ↦ Abandoned
+
+BIPs that had attained Proposed status, i.e. that had been recommended for adoption, may be moved to Abandoned per
+the author’s announcement to the Bitcoin Development Mailing List after not being opposed within 28 days. To successfully
+oppose the move, at least one of the opposers must become the new BIP author for the BIP to remain Proposed. A BIP
+can also be moved to Abandoned by the community if it has had Proposed status for at least one year, there is no
+evidence of it being in active use, and its authors do not object or fail to respond, unless a new author volunteers
+within four weeks.
```

Comment:
```suggestion
BIPs that had attained the Proposed status, i.e. that had been recommended for adoption, may be moved to Abandoned per
the author’s announcement to the Bitcoin Development Mailing List after not being opposed within 28 days. To successfully
oppose the move, at least one of the opposers must become the new BIP author for the BIP to remain Proposed. A BIP
can also be moved to Abandoned by the community if it has had Proposed status for at least one year, there is no
evidence of it being in active use, and its authors do not object or fail to respond, unless a new author volunteers
within four weeks.
```

28 days or 4 weeks? I know it's the same, but maybe pick one for consistency.

Reply:
Sure, four weeks for both.